guides/v2.2/cloud/project/project-conf-files_services.md

```diff
@@ -6,96 +6,139 @@ functional_areas:
+
+-  [`mysql`]({{ page.baseurl }}/cloud/project/project-conf-files_services-mysql.html)
+-  [`redis`]({{ page.baseurl }}/cloud/project/project-conf-files_services-redis.html)
+-  [`elasticsearch`]({{ page.baseurl }}/cloud/project/project-conf-files_services-elastic.html)
+-  [`rabbitmq`]({{ page.baseurl }}/cloud/project/project-conf-files_services-rabbit.html)
 
-A Cloud template includes the following [default `services.yaml` file](https://github.com/magento/magento-cloud/blob/master/.magento/services.yaml):
+You can view default versions and disk values in the the current, [default `services.yaml` file](https://github.com/magento/magento-cloud/blob/master/.magento/services.yaml). The following sample shows the `mysql`, `redis`, and `elasticsearch` services defined in the `services.yaml` configuration file:
 
 ```yaml
 mysql:
-    type: mysql:10.0
+    type: mysql:<version>
     disk: 2048
 
 redis:
-    type: redis:3.0
+    type: redis:<version>
 
 elasticsearch:
-    type: elasticsearch:6.5
+    type: elasticsearch:<version>
     disk: 1024
 ```
 
-Modify this file to use specific and additional services in your deployment. See the [`type`](#cloud-yaml-services-type) section to see the services we support and deploy for you if you add them to the file.
+## Service values
 
-## Service values {#services}
-
-To add a service, you add the following data to services.yaml:
+Minimally, you must provide a `name` and `type` value for each service. If the service uses persistent storage, then you must provide a `disk` value. Use the following format:
 
 ```yaml
-name:
-    type: name:version
-    disk: value
+<name>:
+    type: <name>:<version>
+    disk: <value-MB>
 ```
 
-For example:
+### `name`
 
-```yaml
-mysql:
-    type: mysql:10.0
-    disk: 2048
-```
+The `name` value identifies the service in the project. You can only use lower case alphanumeric characters: `a` to `z` and `0` to `9`, such as `redis`.
 
-### `name` {#cloud-yaml-services-name}
+This name value is used in the `relationships` property of the `.magento.app.yaml` configuration file:
 
-Identifies the service in the project. The `name` can consist only of lower case alphanumeric characters: `a` to `z` and `0` to `9`. For example, Redis is entered as `redis`.
+```yaml
+relationships:
+    redis: "<name>:redis"
+```
 
-You can have multiple instances of each service type. For example, you could have multiple Redis instances. For example, we use multiple Redis instances, one for session and one for cache.
+You can name multiple instances of each service type. For example, we could use multiple Redis instances—one for session and one for cache.
 
 ```yaml
 redis:
-    type: redis:3.0
+    type: redis:<version>
 
 redis2:
-    type: redis:3.0
+    type: redis:<version>
 ```
 
-Be aware, if you rename a service in the `services.yaml` file, the following is **permanently removed**:
+Renaming a service in the `services.yaml` file **permanently removes** the following:
```

Comment:
Maybe make this a warning note since you can potentially lose important data if you miss this.

Reply:
It is difficult to provide a detailed list with a warning.